activerecord-sqlserver-adapter.gemspec

```diff
@@ -28,5 +28,5 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "activerecord", "~> 8.1.0.alpha"
-  spec.add_dependency "tiny_tds", "~> 3"
+  spec.add_dependency "tiny_tds", "~> 3.1"
```

Comment:
Should this not be `3.0`? Basically any v3 release.

Reply:
true, updated the commit.